README.md

```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
 
 Axon is compatible with [Ethereum](https://ethereum.org) so that all of the develop utilities can be used on it directly. And the [Axon CLI](https://github.com/axonweb3/axon-cli) provides an all-in-one client which includes initialization, DevOps, cross-chain request and so on. Seeing is believing, there is a [15 minutes tutorial](https://docs.axonweb3.io/getting-started/for-dapp-devs/zero_to_axon_with_axon_cli) that will lead you to build your own chain and deploy a tiny application.
 
-### Native Cross-chain Communication
+### Native Cross-Chain Communication
 
 Openness and mobility are the foundation of social development, so is blockchain. Cross-chain function enhances liquidity for the web3 ecosystem. Axon develops native cross-chain communication without any bridge. Each Axon-based chain can speak to [CKB](https://www.nervos.org), other Axon-based chains, and any [IBC](https://ibcprotocol.org) compatible chains. Axon will embed more cross-chain protocols in the future.
 
+### Staking on Layer 1 CKB
+
+An Axon-based chain is a Proof-of-Stake (PoS) network. Native AT token holders can stake their tokens to become validators and/or delegate their tokens to other validators to earn rewards. Different from most other sidechains whose staking happens on the layer 2 level, Axon’s staking mechanism takes place on the layer 1 CKB. Validators and other participants stake their native AT tokens on CKB, which uses a Proof-of-Work (PoW) consensus mechanism. This unique staking design helps Axon-based sidechains enjoy the highest degree of decentralization and security from the layer 1 while maintain their high performance and sovereignty as independent layer 2 networks. 
```

Comment:
```suggestion
An Axon-based chain is a Proof-of-Stake (PoS) network. Native AT token holders can stake their tokens to become validators and/or delegate their tokens to other validators to earn rewards. Different from most other sidechains whose staking happens on the layer 2 level, Axon’s staking mechanism takes place on the layer 1 CKB. Validators and other participants stake their native AT token which is an xUDT corresponding to the Axon-based chain on CKB, which uses a Proof-of-Work (PoW) consensus mechanism. This unique staking design helps Axon-based sidechains enjoy the highest degree of decentralization and security from the layer 1 while maintain their high performance and sovereignty as independent layer 2 networks. 
```

Reply:
An Axon-based chains is a Proof-of-Stake (PoS) network. Holders of the Native AT token—Axon't utility token—can stake their tokens to become validators or delegate their tokens to other validators in exchange for rewards. Unlike other sidechains where staking takes place on layer 2, Axon's staking is grounded on layer 1 CKB. Validators and other participants stake their native AT tokens, which  are xUDTs (Extensible User Defined Tokens). The consensus mechanism used by CKB is Proof-of-Work (PoW). This unique staking design allows Axon-based sidechains to enjoy the highest level of decentralization and security from layer 1 while maintaining their high performance and sovereignty as independent layer 2 networks.

Reply:
Comments on the above comments:
Explain "AT token" when it was introduced for the first time.
"AT token" can be defined as "utility token" that allows holders to participate in consensus, rather than "value token".
Add the full name of xUDT.